webui/module/Restore/src/Restore/Controller/RestoreController.php

```diff
@@ -550,9 +550,10 @@ private function buildSubtree()
     $this->setRestoreParams();
     $this->getJobIds();
 
-    if($this->restore_params['id'] !== null && $this->restore_params['id'] !== "#") {
+    if($this->restore_params['id'] == null || $this->restore_params['id'] == "#") {
```

Comment:
you cannot sanely compare with null in PHP. You'll have to use `is_null()` to check if a value is null.
Question is: do you mean `is_null($this->restore_params['id'])`, `array_key_exists('id', $this->restore_params)` or maybe even both?

Reply:
$this->restore_params['id'] is always set and its default value is null, there is no need to check if it exists.
Values are mixed type, integer, string or null.

is_null($a) is same as $a === null whereas $a === null is bit faster than is_null($a) for saving one function call, but I don't want to argue about micro optimization here.

I've corrected the comparison operators to take the types into account.